images/miq-app/ose-assets/appliance-initialize.service

```diff
@@ -1,13 +1,13 @@
 [Unit]
 Description=Initialize Appliance Database
-After=evminit.service memcached.service
+After=evminit.service
 Before=evmserverd.service
-Requires=memcached.service
 [Service]
 TimeoutStartSec=10m
 Type=oneshot
+EnvironmentFile=/container.env.vars
```

Comment:
@fvillain In regards of env variables check in appliance-initialize.sh, if those checks are to support docker-compose as a contrib, we can leave them in place and perhaps put a comment on appliance-initialize.sh to clarify why.